agent/bench-scripts/postprocess/user-benchmark-wrapper

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+benchmark_run_dir=$1
+benchmark_bin=$2
+
+# Run the benchmark and record the duration
+SECONDS=0
+$benchmark_bin 2>&1 | tee $benchmark_results_dir/result.txt
+benchmark_duration=$SECONDS
+
+# Checks if the duration is provided by callee
+if [[ -f "$benchmark_run_dir/user-benchmark-summary.json" ]] && grep -qw "duration" "$benchmark_run_dir/user-benchmark-summary.json"; then
```

Comment:
Can you turn this logic into a python program which does it all?  In addition, it would also verify if the JSON is well formed.